lmp/jobserv.yml

```diff
@@ -356,6 +356,10 @@ triggers:
 
   - name: Code Review
     type: github_pr
+    webhooks:
+      - url: https://conductor.infra.foundries.io/api/lmp/
+        only_failures: false
+        secret_name: lava-webhook-key
```

Comment:
I'd use a more random secret name.

Reply:
it's the name of the variable. Actual secret is not shared in this file.